.gitignore

```diff
@@ -22,4 +23,4 @@ npm-debug.log
 yarn-debug.log
 yarn-error.log
```

Comment:
Do we need all this yarn and stuff in here?

Reply:
These are autocreated I'm happy to remove if you want?